### PR TITLE
Add right image for logsearchapi

### PR DESCRIPTION
**Pull request by cesnietor**
previous image was invalid.
Tested on k8s environment.

### CODE REVIEW

File: restapi/admin_tenants.go [hunk 1]
```diff
@@ -887,7 +887,7 @@ func getTenantCreatedResponse(session *models.Principal, params admin_api.Create
 	}
 	// default activate lgo search and prometheus
 	minInst.Spec.Log = &miniov2.LogConfig{
-		Image: "minio/logsearch:v4.0.0",
```
Comment (harshavardhana) [inline restapi/admin_tenants.go:890]: Will fix this for now.

Comment (cesnietor) [reply restapi/admin_tenants.go:890]: so both will exist?

Comment (harshavardhana) [reply restapi/admin_tenants.go:890]: Only for now @cesnietor until we fix this mistake :-)

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by Alevsk — APPROVED** [review]

**Pull request merged**